pkg/daemon/daemon.go

```diff
@@ -1328,7 +1328,7 @@ type pipeErrorHandler struct {
 	pipe chan error
 }
 
-func (e *pipeErrorHandler) handle(err error) {
+func (e *pipeErrorHandler) handle(_ context.Context, err error, _ string, _ ...interface{}) {
```

Comment:
question: Was this change needed here? I suspect this might be something that got caught between branches.

Reply:
yeah it was :/ it was needed post bump. utilruntime.ErrorHandlers expects a value of a specific type, and the function (&pipeErrorHandler{errCh}).handle did not match the expected type